sdk/tests/deqp/functional/gles3/es3fShaderStateQueryTests.js

```diff
@@ -1089,25 +1089,45 @@ es3fShaderStateQueryTests.VertexAttributeStrideCase.prototype.test = function()
     // Test with default VAO
 
     for (var ndx = 0; ndx < pointers.length; ++ndx) {
-        gl.vertexAttribPointer(0, pointers[ndx][0], pointers[ndx][1], false, pointers[ndx][2], pointers[ndx][3]);
-        this.check(glsStateQuery.verifyVertexAttrib(0, gl.VERTEX_ATTRIB_ARRAY_STRIDE, pointers[ndx][2]));
+        try {
+            gl.vertexAttribPointer(0, pointers[ndx][0], pointers[ndx][1], false, pointers[ndx][2], pointers[ndx][3]);
+        } catch (err) {
```

Comment:
This try/catch mechanism for catching the GL errors only works because sdk/tests/deqp/functional/gles3/shaderstatequery.html is using wtu.create3DContextWithWrapperThatThrowsOnGLError. It's unfortunate that this causes a significant change in the control flow, especially because at some point we'll likely want to turn off that context wrapper. I'll merge this now and look into how this can be cleaned up.